### DISCUSSION TITLE
Querying over local HDT files

### DISCUSSION THREAD

**Discussion by balhoff** [comunica/comunica#1314] [Q&A] [answered]
Is it currently possible to run SPARQL queries over local HDT files? I saw a comment in https://comunica.dev/docs/query/advanced/hdt/ that this functionality needs to be updated. Then I saw that this repo was just recently created: https://github.com/comunica/comunica-feature-hdt I tried to clone it and run locally but I still get these errors:

```
All actors rejected their test in urn:comunica:default:query-source-identify-hypermedia/mediators#main
Actor urn:comunica:default:query-source-identify-hypermedia/actors#qpf is not able to handle source type hdtFile.
Actor urn:comunica:default:rdf-resolve-quad-pattern/actors#sparql is not able to handle source type hdtFile.
Actor urn:comunica:default:query-source-identify-hypermedia/actors#none is not able to handle source type hdtFile.
```

Thanks!

**Comment by rubensworks** [comment]
The new repo is still a WIP, but it's being worked on. I don't expect it to take very long to finish.

In the meantime, `@comunica/actor-init-sparql-hdt` should be usable, but may only work on older Node.js versions.

**Reply by rubensworks** [reply, marked as answer]
`@comunica/query-sparql-hdt` has been released: https://www.npmjs.com/package/@comunica/query-sparql-hdt

**Reply by balhoff** [reply]
Excellent, thanks so much.
